lib/web_ui/lib/src/engine/html/dom_canvas.dart

```diff
@@ -161,10 +160,10 @@ ui.Color blurColor(ui.Color color, double sigma) {
   return ui.Color((reducedAlpha & 0xff) << 24 | (color.value & 0x00ffffff));
 }
 
-html.HtmlElement buildDrawRectElement(
+DomHTMLElement buildDrawRectElement(
     ui.Rect rect, SurfacePaintData paint, String tagName, Matrix4 transform) {
-  final html.HtmlElement rectangle =
-      html.document.createElement(tagName) as html.HtmlElement;
+  final DomHTMLElement rectangle = domDocument.createElement(tagName) as
+      DomHTMLElement;
```

Comment:
Is the `as DomHTMLElement` necessary here? AFAICT you can create either HTML or SVG elements, and to create SVG you need to use `createElementNS`. So maybe `domDocument.createElement` should be typed as `DomHTMLElement`?

Reply:
So the MDN says: A new HTMLElement is returned if the document is an HTMLDocument, which is the most common case. Otherwise a new Element is returned.

In all likelihood, it should be safe to type `createElement` with DomHTMLElement, but `dart:html` typed it with `Element` and I followed the pattern of `dart:html` out of fear of breaking something in a strange case.

If you don't think there is any possibility where this could suddenly return an `Element` on any browser we care about then we can change the type. Thoughts?